src/blister.h

```diff
@@ -52,7 +52,7 @@ static inline Int IS_BLIST_REP(Obj list)
 
 static inline Int IS_BLIST_REP_WITH_COPYING(Obj list)
 {
-    Int tnum = TNUM_OBJ(list);
+    UInt tnum = TNUM_OBJ(list);
```

Comment:
Commit message is "Treat TNUMs are UInts, not Ints" -> "are" should b "as", I guess?